i18n/rs.po

```diff
@@ -142,13 +142,13 @@ msgid "News"
 msgstr "Vesti"
 
 msgid "How do I <i>use</i> pool.ntp.org?"
-msgstr "Kako da <i>use</i> pool.ntp.org?"
+msgstr "Kako da <i>koristim</i> pool.ntp.org?"
 
 msgid "How do I <i>join</i> pool.ntp.org?"
-msgstr "Kako da se pridružim <i>join</i> pool.ntp.org?"
+msgstr "Kako da se <i>pridru�im</i> pool.ntp.org?"
```

Comment:
Hi -- can you double-check these? It looks like the character encoding got messed up.